scripts/CSSRE/under-replicated-partitions/under-replicated-partitions.py

```diff
@@ -0,0 +1,57 @@
+import argparse
+import sys
+
+LIB_PATH = "/managed-scripts/CSSRE/lib"
+# Use sys.path.insert because PYTHONPATH's support is unclear
+sys.path.insert(0, LIB_PATH)
+
+from py.script import Script
+from py.kafka import Kafka
```

Comment:
I'm seeing:
```
Traceback (most recent call last):
  File "<string>", line 9, in <module>
ModuleNotFoundError: No module named 'py.kafka'
```
but im also unsure why and not sure if my hack at a testjob is the problem.
The pod is `openshift-job-dev-x79qh`